notebooks/byod.py

```diff
@@ -186,7 +192,7 @@ def create_cram_crai_table(table: str, listing: Iterable[str]):
     """
 
 with callysto.Cell("python"):
-    fulldir = bucket+"/"+subdirectory
+    fulldir = bucket + "/" + subdirectory
```

Comment:
Consider asking the user to list the path that they uploaded, which may or may not be "subdirectory".

Reply:
It already does, see line 162. All that changed here from #20 is that on #27 flake8 decided that spaces are needed between operators.